Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/response/TestInboundPatientDiscoveryDeferredResponse.java

```diff
@@ -37,8 +37,16 @@
  */
 public class TestInboundPatientDiscoveryDeferredResponse implements InboundPatientDiscoveryDeferredResponse{
```

Comment:
Do we need a whole class to do just this?  Would it not be better to in-line override a different class that implements InboundPatientDiscoveryDeferredResponse, or to mock the InboundPatientDiscoveryDeferredResponse to return a new MCCIIN000002UV01 when ".respondingGatewayDeferredPRPAIN201306UV02()" is called?